src/pages/components/notification/usage.mdx

```diff
@@ -4,79 +4,198 @@ description: Notifications are messages that communicate information to the user
+![Example of inline notification placement](images/notification-placement-inline.png)
+
+<Caption>
+  Example of an inline notification supporting an error message on a form
+</Caption>
 
 </Column>
 </Row>
 
-## Placement
+### Behavior
 
-#### Toast notifications
+#### Dismissal
 
-Toast notifications slide in and out a page from the top-right corner. Actionable notifications do not appear on mobile screen widths.
+Inline notifications do not dismiss automatically. They persist on the page until the user dismisses them or takes action that resolves the notification.
+
+A small "x" in the top right corner is used to dismiss inline notifications. Including the close button is optional and should not be included if it is critical for a user to read or interact with the notification.
+
+## Toast notifications
+
+### Formatting
 
 <Row>
 <Column colLg={8}>
 
-![Toast notification example](images/notification-usage-2.png)
+![Example of toast notification with action button and time stamp](images/notification-toast-action.png)
 
 </Column>
 </Row>
 
-#### Inline notifications
+#### Action
+
+Toast notifications can include a link at the end of their body content. This link should be short and navigate users to a page or modal where they can take action to address the notification or find further information. Because toast notifications automatically dismiss, there should be alternative routes to navigate to the link destination.
+
+If you include links within the notification body that redirect the user to steps, be sure keyboard and screenreader users have enough time to access the notification or can easily find the notification after it disappears.
+
+#### Time stamp
+
+Toast notifications include a time stamp at the bottom the container. The time stamp shows the time the notification was sent. Using time stamps is optional but toasts should be consistent across the product so either all toasts should include time stamps or none of them should. The time stamp is optional and can be removed if a third line of content is needed.
+
+#### Sizing
 
-Inline notifications appear near its related item. In [forms](/components/form/code), we recommend placing the inline notification at the bottom of the form, right before the submission buttons. Depending on the context of the page, inline notifications can appear above the content as well.
+Toast notifications use a fixed width and their height depends on the length of the notification message. As noted in the content guidelines, limit toast notifications to two lines of text.
+
+#### Placement
+
+Toast notifications slide in and out from the top right of the screen. They align to the edge of the last column and can stack with `$spacing-03` in-between. New toast notifications should appear at the top of the list, with older notifications being pushed down until they are dismissed.
+
+Actionable toast notifications do not appear on mobile screen widths.
```

Comment:
What was the reasoning behind this constraint? 👀

Reply:
This is existing guidance we have right now. I'm not sure the original reasoning behind it but I'm trying not to changing existing guidance without having a pretty good reason to change it.

Reply:
Got it! Makes sense 👍